ingress/nginx.tmpl

```diff
@@ -0,0 +1,44 @@
+worker_processes  1;
+daemon off;
+
+error_log  /dev/stdout warn;
+pid        {{ .WorkingDir }}/nginx.pid;
+
+events {
+ worker_connections  1024;
+}
+
+http {
+ default_type  application/octet-stream;
+
+ log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+ '$status $body_bytes_sent "$http_referer" '
+ '"$http_user_agent" "$http_x_forwarded_for"';
+
+ access_log  /var/log/nginx/access.log  main;
+ sendfile        on;
+ keepalive_timeout  65;
+
+ {{ $port := .Port }}
+ {{ range $entry := .Entries }}
+ # Start entry
+ server {
+   listen {{ $port }};
+   server_name {{ $entry.Host }};
+   location {{ $entry.Path }} {
+     proxy_pass http://{{ $entry.ServiceName }}:{{ $entry.ServicePort }};
+   }
+ }
+ # End entry
+ {{ end }}
+
+ # default server for health checks
+ server {
+   listen {{ .Port }} default_server;
+   location /health {
+     access_log off;
+     return 200;
+   }
+ }
+
+}
```

Comment:
Few more things:
- Didn't we get an error message about the hash bucket size being too small? I think the default is too small for the xeon cpus we use on AWS. But not sure.
- I'd rather start with an optimized config from online (such as the one in kubernetes contrib), rather than sticking with nginx defaults - which afaik are not great out of the box.